third-party-synthetic/active-gate-extensions/extension-third-party-ping/src/plugin.json

```diff
@@ -1,131 +1,145 @@
 {
-    "name": "custom.remote.python.thirdparty_ping",
-    "version": "1.008",
-    "type": "python",
-    "entity": "CUSTOM_DEVICE",
-    "technologies": ["ICMP"],
-    "metricGroup": "tech.Ping",
-    "source": {
-        "package": "ping_extension",
-        "className": "PingExtension",
-        "install_requires": ["pingparsing", "dtapi"],
-        "activation": "Remote"
+  "name": "custom.remote.python.thirdparty_ping",
+  "version": "1.2",
+  "type": "python",
+  "entity": "CUSTOM_DEVICE",
+  "technologies": [ "ICMP" ],
+  "metricGroup": "tech.Ping",
+  "source": {
+    "package": "ping_extension",
+    "className": "PingExtension",
+    "install_requires": [ "pingparsing", "dtapi" ],
+    "activation": "Remote"
+  },
+  "properties": [
+    {
+      "key": "api_url",
+      "type": "String"
     },
+    {
+      "key": "api_token",
+      "type": "Password"
+    },
+    {
+      "key": "test_name",
+      "type": "String"
+    },
+    {
+      "key": "test_target",
+      "type": "String"
+    },
+    {
+      "key": "test_location",
+      "type": "String"
+    },
+    {
+      "key": "log_level",
+      "type": "Dropdown",
+      "dropdownValues": [ "INFO", "DEBUG" ]
+    },
+    {
+      "key": "frequency",
+      "type": "String"
+    },
+    {
+      "key": "proxy_address",
+      "type": "String"
+    },
+    {
+      "key": "proxy_username",
+      "type": "String"
+    },
+    {
+      "key": "proxy_password",
+      "type": "Password"
+    },
+    {
+      "key":  "failureCount",
```

Comment:
@florent-duchateau please follow the same format as the other keys, `failure_count` for consistency